plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkModifyJob.java

```diff
@@ -64,7 +64,7 @@ public CloudSdkModifyJob(String jobName, MessageConsoleStream consoleStream,
 
   @VisibleForTesting
   MessageConsoleStream createNewMessageConsole() {
-    MessageConsole console = MessageConsoleUtilities.getMessageConsole(
+    MessageConsole console = new MessageConsoleUtilities().getMessageConsole(
```

Comment:
constantly creating an object, calling one method on it, and then letting the object be garbage collected strongly suggests you really want a static method

Reply:
That was for testing. I'll revert them to `static` and take a different approach to inject a `IConsoleManager`.